docs/source/guides/writer/chapters/writing.rst

```diff
@@ -790,6 +792,10 @@ namely the ``--output-check-record`` argument with values ``stdout``,
 to the files ``stdout.expected`` and ``stderr.expected`` at the test's
 data directory (which is different from the job/test results directory).
 
+.. note:: The ``--output-check-record`` is legacy feature and
```

Comment:
IMO, for consistency and easier reading of the documentation, we should remove the paragraph above, rather than adding this note.

Reply:
sure, make sense